build_runner/lib/src/server/server.dart

```diff
@@ -18,14 +18,14 @@ Future<HttpServer> _futureServer;
 Handler blockingHandler;
 
 /// Starts a server which blocks on any ongoing builds.
-Future<HttpServer> startServer(BuildResults results, BuildOptions options) {
+Future<HttpServer> startServer(BuildState buildState, BuildOptions options) {
   if (_futureServer != null) {
     throw new StateError('Server already running.');
   }
 
   try {
     blockingHandler = (Request request) async {
-      if (results.currentBuild != null) await results.currentBuild;
+      if (buildState.currentBuild != null) await buildState.currentBuild;
```

Comment:
I think you can await null, so you probably don't need this if?

Reply:
Done.